config/webhook/service.yaml

```diff
@@ -0,0 +1,19 @@
+apiVersion: v1
+kind: Service
+metadata:
+  labels:
+    app.kubernetes.io/name: service
+    app.kubernetes.io/instance: webhook-service
+    app.kubernetes.io/component: webhook
+    app.kubernetes.io/created-by: k0smotron
+    app.kubernetes.io/part-of: k0smotron
+    app.kubernetes.io/managed-by: kustomize
+  name: webhook-service
```

Comment:
Feels bit too generic name for the webhoook svc as this is specific to k0smotron

Reply:
It's a base name, that will be patched in kustomize to `k0smotron-webhook-service` in standalone mode and to `k0smotron-webhook-service-control-plane` for CAPI.